### DISCUSSION TITLE
一看冠以学之名我以为有什么高见，没想到全是一些低劣的思想

### DISCUSSION THREAD

**Discussion by LgnMs** [The-Run-Philosophy-Organization/run#600] [General]
给我的一天增加了笑料

**Comment by LgnMs** [comment]
我一开始想知道的就是润和没润到底有什么区别，但是一打开文章全是充满偏见毫无逻辑的一家之言，更能强烈感受到作者面对布尔什维克的无能狂怒。你可以说人是腐败的，但是就靠那些低劣无逻辑的言论就能抹黑了吗，如果是这样剑桥五杰为何会是最坚实的共产主义信仰者

**Reply by alexdbug** [reply]
真的难绷。通篇读下来最后作者的结论是布尔什维克不行

**Reply by seekCat969** [reply]
那你告诉我 先锋队的腐化怎么解决？当年也有不少知识分子带着浪漫滤镜跑去延安哦 最后下场可不怎么样哦😯
